src/main/java/seedu/address/model/person/Github.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.model.person;
+
+/**
+ * Represents a person's Github details in JABPro.
+ */
+public class Github {
+
+    public final String value;
```

Comment:
Should not be null , need to refer to other fields that was incorporated into person like Address, phone number.